dnanexus/src/eggd_athena.sh

```
@@ -55,8 +63,7 @@ main() {
     dx-download-all-inputs --parallel
 
     per_base_coverage=$(find /home/dnanexus/in/coverage_files -type f -name "*.per-base.bed.gz")
-    build_file=$(find /home/dnanexus/in/coverage_files -name "*build.txt")
-    [ -n "$build_file" ] && build="--build $(cat $build_file)"
+    reference_file=$(find /home/dnanexus/in/coverage_files -name "*build.txt")
```

Shellcheck (0.10.0)

[warning] reference_file appears unused. Verify use (or export if used externally).

(SC2034)